package.json

```diff
@@ -97,8 +93,6 @@
     "mock-mail-adapter": "file:spec/dependencies/mock-mail-adapter",
     "mongodb-runner": "4.8.1",
     "mongodb-version-list": "1.0.0",
-    "node-fetch": "3.2.10",
```

Comment:
Same for this and `AbortController`.

Reply:
And this isn't used anywhere else in the codebase?

Reply:
No it isn't. It is used only in tests so it is a dev dependency.